example_app.yml

```diff
@@ -0,0 +1,103 @@
+## see https://meta.discourse.org/t/17247 for details
+expose:
+  - "8080:80"   # http
+  - "2222:22"
+  - "8443:443" # https
+
+params:
+  db_default_text_search_config: "pg_catalog.english"
+
+  ## Set db_shared_buffers to a max of 25% of the total memory.
+  ## will be set automatically by bootstrap based on detected RAM, or you can override
+  #db_shared_buffers: "256MB"
+
+  ## can improve sorting performance, but adds memory usage per-connection
+  #db_work_mem: "40MB"
+
+  ## Which Git revision should this container use? (default: tests-passed)
+  #version: tests-passed
+
+env:
+  LANG: en_US.UTF-8
+  # DISCOURSE_DEFAULT_LOCALE: en
+
+  ## How many concurrent web requests are supported? Depends on memory and CPU cores.
+  ## will be set automatically by bootstrap based on detected CPUs, or you can override
+  #UNICORN_WORKERS: 3
+
+  ## Required. Discourse will not work with a bare IP number.
+  DISCOURSE_HOSTNAME: 'lvh.me:8443'
+
+  ## Uncomment if you want the container to be started with the same
+  ## hostname (-h option) as specified above (default "$hostname-$config")
+  #DOCKER_USE_HOSTNAME: true
+
+  ## TODO: List of comma delimited emails that will be made admin and developer
+  ## on initial signup example 'user1@example.com,user2@example.com'
+  DISCOURSE_DEVELOPER_EMAILS: 'dan@mutations.ltd'
+
+  ## TODO: The SMTP mail server used to validate new accounts and send notifications
+  # SMTP ADDRESS, username, and password are required
+  # WARNING the char '#' in SMTP password can cause problems!
+  # smtp.mailtrap.io is an example that can be used.
+  DISCOURSE_SMTP_ADDRESS: smtp.mailtrap.io
+  #DISCOURSE_SMTP_PORT: 587
+  DISCOURSE_SMTP_USER_NAME: TODO: SMTP USER NAME
+  DISCOURSE_SMTP_PASSWORD: TODO: SMTP PASSWORD
+  #DISCOURSE_SMTP_ENABLE_START_TLS: true           # (optional, default true)
+
+  ## If you added the Lets Encrypt template, uncomment below to get a free SSL certificate
+  # LETSENCRYPT_ACCOUNT_EMAIL: me@lvh.me
+
+  ## The http or https CDN address for this Discourse instance (configured to pull)
+  ## see https://meta.discourse.org/t/14857 for details
+  #DISCOURSE_CDN_URL: https://discourse-cdn.example.com
+
+## The Docker container is stateless; all data is stored in /shared
+volumes:
+  - volume:
+      host: TODO: update path /discourse_docker/shared/standalone
+      guest: /shared
+  - volume:
+      host: TODO: update path /discourse_docker/shared/standalone/log/var-log
+      guest: /var/log
+
+## Plugins go here
+## see https://meta.discourse.org/t/19157 for details
+hooks:
+  after_code:
+    - exec:
+        cd: $home/plugins
+        cmd:
+          - mkdir -p plugins
+          - git clone https://github.com/discourse/docker_manager.git
+          - git clone https://github.com/mutations/discourse-patreon-creator.git
+          # - git clone -b db/feature--wording-change--169689677 https://github.com/mutations/discourse-patreon-creator.git
```

Comment:
This looks like it could/should maybe be removed.

Reply:
@phlipper I was leaving that there as an example of how to specify a feature branch.